workflow/scripts/osemosys_global/summary/costs.py

```diff
@@ -47,6 +47,42 @@ def get_storage_cost(
         .sum()
     )
 
+def get_transmission_cost(discounted_cost_tech: pd.DataFrame, country: bool) -> pd.DataFrame:
+
+    df = discounted_cost_tech.copy()
+
+    df1 = df.copy()[
+        (df.index.get_level_values("TECHNOLOGY").str.startswith("TRN"))
+    ]
+
+    if country:
+        r = "COUNTRY"
+        df1[r] = df1.index.get_level_values("TECHNOLOGY").str[3:6]
+    else:
+        r = "NODE"
+        df1[r] = df1.index.get_level_values("TECHNOLOGY").str[3:8]
+    
+    df2 = df.copy()[
+        (df.index.get_level_values("TECHNOLOGY").str.startswith("TRN"))
+    ]
+
+    if country:
+        r = "COUNTRY"
+        df2[r] = df2.index.get_level_values("TECHNOLOGY").str[8:11]
+    else:
+        r = "NODE"
+        df2[r] = df2.index.get_level_values("TECHNOLOGY").str[8:13]
+        
+    for df in [df1, df2]:
+        df['VALUE'] = df['VALUE'] / 2
+        
+    df = pd.concat([df1, df2])
+
+    return (
+        df.reset_index()[["REGION", r, "YEAR", "VALUE"]]
+        .groupby(["REGION", r, "YEAR"])
+        .sum()
+    )
```

Comment:
I find grouping the logic in the `if` `else` statement together easier to read, but Im not fussed either way. Here is a suggestion if you do want to apply it. :) 

```suggestion
def get_transmission_cost(discounted_cost_tech: pd.DataFrame, country: bool) -> pd.DataFrame:

    cost = discounted_cost_tech[
        (discounted_cost_tech.index.get_level_values("TECHNOLOGY").str.startswith("TRN"))
    ].copy()
    
    df1 = cost.copy()
    df2 = cost.copy()

    if country:
        r = "COUNTRY"
        df1[r] = df1.index.get_level_values("TECHNOLOGY").str[3:6]
        df2[r] = df2.index.get_level_values("TECHNOLOGY").str[8:11]
    else:
        r = "NODE"
        df1[r] = df1.index.get_level_values("TECHNOLOGY").str[3:8]
        df2[r] = df2.index.get_level_values("TECHNOLOGY").str[8:13]

    for df in [df1, df2]:
        df['VALUE'] = df['VALUE'] / 2
        
    df = pd.concat([df1, df2])

    return (
        df.reset_index()[["REGION", r, "YEAR", "VALUE"]]
        .groupby(["REGION", r, "YEAR"])
        .sum()
    )
```